Dockerfile.assisted-service

```diff
@@ -1,35 +1,26 @@
+# This file was automatically generated by `hack/generate.sh generate_dockerfiles`
+# PLEASE DO NOT MODIFY THIS FILE. Modify Dockerfile.assisted-service.jinja2 instead,
+# then rerun the above command.
+
 # Generate python client
 FROM quay.io/ocpmetal/swagger-codegen-cli:2.4.15 as swagger_py
 
 COPY swagger.yaml .
 COPY tools/generate_python_client.sh .
 RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
 
-# Generate go client
-FROM quay.io/goswagger/swagger:v0.25.0 as swagger_go
-COPY swagger.yaml go.* ./
-RUN mkdir /build
-RUN swagger generate server --template=stratoscale -f swagger.yaml --template-dir=/templates/contrib \
-    && mv restapi models /build
-RUN swagger generate client --template=stratoscale -f swagger.yaml --template-dir=/templates/contrib \
-    && mv client /build
+FROM registry.ci.openshift.org/openshift/release:golang-1.15 as pybuilder
```

Comment:
The go client should still be built. We shouldn't name a golang base image pybuilder.

Reply:
Sorry for splitting the discussion, see my response in the other PR regarding the